GeneralStateTests/stCreateTest/CREATE_EOF1_FromEOF.json

```diff
@@ -0,0 +1,216 @@
+                },
+                {
+                    "hash" : "0x9f159b36826ec1d9763f06e6202a52f064fe514b26567ed25b693f1045e57cac",
+                    "indexes" : {
+                        "data" : 9,
+                        "gas" : 0,
+                        "value" : 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+                    "txbytes" : "0xf895800a83e4e1c094b94f5374fce5edbc8e2a8697c15331677e6ebf0b80b5ef0001010029020002007fef000101000302000400600000aabbccdd00000000000000000000000000000060005260116000f3eeff1ba0b2c0b77f46b143420d45cbb17a667f1005bda0dd8f9cdb1aaa92d0c8b8b65f5ca061b43cbe4f0dd9a6c8790e0b28725bdad060bf89021c1b3920e0f999e94ad4c8"
+                },
+                {
+                    "hash" : "0xdda43b820582d40491b8ffce169d224a1cf98c38819b764f5d67dd7bb6180197",
+                    "indexes" : {
+                        "data" : 10,
+                        "gas" : 0,
+                        "value" : 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+                    "txbytes" : "0xf86c800a83e4e1c094b94f5374fce5edbc8e2a8697c15331677e6ebf0b808cef00010100050060016000f31ca0698592d3ca907f71cabf49e1f6332cbc244774b4268155b547d7f2a783084a0aa0088b110fdc38b542dbb422ef67e247e191c2d0c851aac20c07553a129ecebc07"
+                },
+                {
+                    "hash" : "0x02ecccd7c497ad5a04df692d2d4d2bb39174bf3d4a010926d9b6953f012aadb8",
+                    "indexes" : {
+                        "data" : 11,
+                        "gas" : 0,
+                        "value" : 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+                    "txbytes" : "0xf873800a83e4e1c094b94f5374fce5edbc8e2a8697c15331677e6ebf0b8093ef00010100050200040060016000f3aabbccdd1ca0eaac7dd6e7a8dce1d277221b1d265a99ab438fdceefbde6d33de6bdf7ac22982a07ee69d3b6dee87615dcde64ba42f05c0a448ba97394ece72242a4e62a110696c"
+                },
+                {
+                    "hash" : "0xc61814b3888f35a254cda924d7515af20203bf3e355d6a6fa119c2800aa9670e",
+                    "indexes" : {
+                        "data" : 12,
+                        "gas" : 0,
+                        "value" : 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+                    "txbytes" : "0xf865800a83e4e1c094b94f5374fce5edbc8e2a8697c15331677e6ebf0b808560016000f31ba08ded9ae1fd850114385f132b1e74d4e123627865bf35a8ccdd1571c4f019b4c4a005536793e1483242976ce12f1a4e9724391d86f23c8d3bad310fa63d636c3ad0"
```

Comment:
According to our implementation, the computed size of the EOF container stored in 0x174f5374fce5edbc8e2a8697c15331677e6ebf0b that is called in this transaction is 126 while the actual bytecode length is 128 so this test is failing in the EthereumJS implementation.

Reply:
This sends transaction to contract `0xb94f5374fce5edbc8e2a8697c15331677e6ebf0b`, which creates contract `0xf1ecf98489fa9ed60a664fc4998db699cfa39d40` 1-byte long
(are you sure problem is in this test file (`CREATE_EOF1_FromEOF`)?)